pravega/pravega-client-rust
# Issue 186: Enable RUST logs from python bindings.

**Change log description**  
Enable RUST logs during invocation of Python bindings.

**Purpose of the change**  
Fixes #186 

**What the code does**  
Ensure the https://docs.rs/tracing crate is used by the bindings crate.

**How to verify it**  
Ensure RUST logs are visibile while executing the python APIs.

```
export RUST_LOG=info
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pravega_client
>>> stream_manager=pravega_client.StreamManager("127.0.0.1:9090")
>>> stream_manager.create_scope(scope)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'scope' is not defined
>>> scope="a"
>>> stream_manager.create_scope(scope)
Dec 14 15:01:35.380  INFO pravega_client::stream_manager: creating scope "a"
Dec 14 15:01:35.405  INFO pravega_client::stream_manager: Scope creation status Ok(true)
True
```
